lib/stack.dart

```diff
@@ -33,4 +33,18 @@ class Stack<T> {
 
   /// get the length of the stack.
   int get length => size();
+  
+  // returns true if element is found at the top of the stack
+  bool find(T x) => (x == _list.last) ? true : false;
+  
+  // returns true if element is found in the stack
```

Comment:
@AunSyedShah 
use `\\\` instead of `\\` for documentation comments

Reply:
doing it thanks.